test/test_linalg.py

```diff
@@ -2086,7 +2089,10 @@ def test_linalg_svd_compute_uv(self, device, dtype):
             # check linalg.svd vs numpy
             expected = np.linalg.svd(np_t, full_matrices, compute_uv=True)
             actual = torch.linalg.svd(t, full_matrices, compute_uv=True)
-            self.assertEqual(actual, expected)
+            # sign/phase of the singular vectors is not unique and therefore absolute values are compared
+            self.assertEqual(abs(actual[0]), abs(expected[0]))
+            self.assertEqual(actual[1], expected[1])
+            self.assertEqual(abs(actual[2][:t.shape[0], :]), abs(expected[2][:t.shape[0], :]))
```

Comment:
Probably we can remove this `[:t.shape[0], :]` indexing. It should work now without it as `full_matrices=True` behavior is fixed now.